pkg/opcfg/config.go

```diff
@@ -28,93 +29,236 @@ import (
 	lumberjack "gopkg.in/natefinch/lumberjack.v2"
 )
 
+// GetIsWebhookEnabled returns true if the webhook is enabled.
+func GetIsWebhookEnabled() bool {
+	return lookupBoolEnvVar("WEBHOOKS_ENABLED", envMustExist)
+}
+
+// GetIsControllersEnabled returns true if the controllers for each custom
+// resource will start. If this is false, then the manager will just act as a
+// webhook (if enabled).
+func GetIsControllersEnabled() bool {
+	return lookupBoolEnvVar("CONTROLLERS_ENABLED", envMustExist)
+}
+
+// GetAreControllersNamespaceScoped returns true if the controllers only watch a
+// single namespace.
+func GetAreControllersNamespaceScoped() bool {
```

Comment:
This function's name sounds strange. Why not just `AreControllersNamespaceScoped`?